content/en/post/spring2021-graduation/index.md

```diff
@@ -0,0 +1,32 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Spring 2021 Graduation"
+subtitle: ""
+summary: ""
+authors: []
+tags: []
+categories: []
+date: 2021-03-23T17:19:14+09:00
+lastmod: 2021-03-23T17:19:14+09:00
```

Comment:
Change these dates to 3/25 since they are officially receiving their degrees tomorrow.